src/components/ChangesetFileList/ChangesetFileList.js

```diff
@@ -68,27 +32,15 @@ const ChangesetFileList = ({ compact, files }: Props) =>
                       added={file.stats.added}
                       changed={0}  // FIXME
                     />
-
                   </Col>
                 }
                 <Col lg={8} md={8} sm={8} xs={8}>
-                  <div>
+                  <Link
+                    style={{ cursor: 'pointer' }}
+                    to={`${window.location.pathname}/${file.name}`}
```

Comment:
No, we don't have URL, they were just anchors, by clicking on file link it just scrolls down to the file diff view on the same page.